CryptoPrices/Sources/MyCoin/Sources/MyCoin/CoinStatisticsSection/CoinStatisticsSection.swift

```diff
@@ -52,30 +60,45 @@ private extension CoinStatisticsSection {
         HStack {
             VStack(
                 alignment: .leading,
-                spacing: 16.0
+                spacing: 10.0
             ) {
                 Text(title)
                     .foregroundColor(Colors.textMedium.swiftUIColor)
                     .font(Fonts.Inter.medium.textStyle(.caption))
 
-                Text(price, format: .dollarCurrency)
-                    .foregroundColor(Colors.titleMedium.swiftUIColor)
-                    .font(Fonts.Inter.medium.textStyle(.callout))
+                ZStack(alignment: .leading) {
+                    Text(price, format: .dollarCurrency)
+                        .foregroundColor(Colors.titleMedium.swiftUIColor)
+                        .font(Fonts.Inter.medium.textStyle(.callout))
+                        .opacity(showData ? 1.0 : 0.0)
+                    
+                    Text(Strings.MyCoin.NoData.text)
+                        .foregroundColor(Colors.titleMedium.swiftUIColor)
+                        .font(Fonts.Inter.medium.textStyle(.callout))
+                        .opacity(showData ? 0.0 : 1.0)
+                }
             }
 
             Spacer()
 
             isPriceUp
-            ? Images.icArrowUpGreen.swiftUIImage
-            : Images.icArrowDownRed.swiftUIImage
+            ? Images.icArrowUpGreen.swiftUIImage.opacity(showData ? 1.0 : 0.0)
+            : Images.icArrowDownRed.swiftUIImage.opacity(showData ? 1.0 : 0.0)
 
-            Text(percentage, format: .percentage)
-                .font(Fonts.Inter.medium.textStyle(.body))
-                .foregroundColor(
-                    isPriceUp
-                    ? Colors.guppieGreen.swiftUIColor
-                    : Colors.carnation.swiftUIColor
-                )
+            ZStack(alignment: .trailing) {
+                Text(percentage, format: .percentage)
+                    .font(Fonts.Inter.medium.textStyle(.body))
+                    .foregroundColor(
+                        isPriceUp
+                        ? Colors.guppieGreen.swiftUIColor
+                        : Colors.carnation.swiftUIColor
+                    )
+                    .opacity(showData ? 1.0 : 0.0)
+                
+                Text(Strings.MyCoin.NoData.text)
+                    .font(Fonts.Inter.medium.textStyle(.body))
+                    .opacity(showData ? 0.0 : 1.0)
+            }
```

Comment:
I don't know whether I like this change or the previous though 😄
Because we replace:
```swift
if let something = something {
  ViewA()
} else {
  ViewB()
}
```
with
```swift
ZStack {
  ViewA()
  .opacity(condition ? 1.0 : 0.0)

  ViewB()
  .opacity(condition ? 0.0 : 1.0)
}
```
along with introducing some _ternary conditional_ operators (it's basically the syntactic sugar for `if ... else` I believe)

Anyway, I think it's just a matter of preference. I really appreciate your effort striking to make the code cleaner, easy to read, and maintainable. And we have many chances to tackle this more seriously later. So let's go 🚀